app-backend/api/src/main/scala/Router.scala

```diff
@@ -62,6 +63,7 @@ trait Router extends HealthCheckRoutes
     } ~
     pathPrefix("api") {
       pathPrefix("projects") { projectRoutes } ~
+      pathPrefix("aoi") { aoiRoutes } ~
```

Comment:
Yeah, I think URLs are generally read more often than typed fully and I don't think there is much advantage to having an abbreviation here so I think it should be switched here and in the spec.